### PR TITLE
More than 100 tweets

### LINKED ISSUE #14: Ensure "filter:safe" is turned on
For the Twitter API search

### LINKED ISSUE #72: Get more than 100 tweets per request from twitter
See http://stackoverflow.com/questions/17887984/is-it-possible-to-get-more-than-100-tweets

### PR DESCRIPTION
**Pull request by Winwardo**
Fixes #72 
Fixes #14 
References #48

### CODE REVIEW

**Comment by Winwardo** [comment]
@VasilyShelkov ready for comment and merge

File: src/client/tweetAnalysis.js [hunk 1]
```diff
@@ -154,9 +154,32 @@ const wordCountToSortedList = (wordCounts) => {
...
+  const tenMinutes = 1000 * 60 * 10;
+  const xMinutesAgo = now - tenMinutes;
+
+  const maxOldPosts = 5;
+
+  let outOfDateCount = 0;
+  for (const result in feedResults.data) {
+    const test = new Date(data.data.date).getTime();
+    if (test < xMinutesAgo) {
+      outOfDateCount++;
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Unexpected use of '++'.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2347668677/issues/source?bid=3148782&fileBranchId=3216643#l175)

File: src/client/tweetAnalysis.js [hunk 2]
```diff
@@ -154,9 +154,32 @@ const wordCountToSortedList = (wordCounts) => {
...
+ * over 10 minutes old, it is considered not relevant.
+ * @param feedResults
+ * @returns {boolean}
+ */
 export const doesFeedHaveUsefulResults = (feedResults) => {
-  // TODO: Make this more useful
-  return feedResults.data.count > 10;
+  const now = new Date().getTime();
+  const tenMinutes = 1000 * 60 * 10;
+  const xMinutesAgo = now - tenMinutes;
```
Comment (VasilyShelkov) [inline src/client/tweetAnalysis.js]: Use moment

File: src/server/twitterSearch.js [hunk 3]
```diff
@@ -261,6 +264,41 @@ export const searchAndSaveFromTwitter = (query, count = 300) => {
...
+const sweepTwitterAndConcat = (query, count, existingStatuses = [], lowestId = null) => {
+  let extendedQuery = query;
+  if (lowestId !== null) {
+    extendedQuery += ` max_id:${lowestId}`;
+  };
+
+  return newPromiseChain()
+    .then(() => potentiallySearchTwitter(extendedQuery, count))
+    .then((twitStatuses) => {
+      const countLeft = count - 100;
```
Comment (VasilyShelkov) [inline src/server/twitterSearch.js]: how comes you take away a static 100 instead of:
`const countLeft = count - added` the added that's below

Comment (Winwardo) [inline src/server/twitterSearch.js]: cause i'm a dumb

**Pull request merged**